daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ScriptF.scala

```diff
@@ -675,6 +683,63 @@ object ScriptF {
       } yield SEAppAtomic(SEValue(continue), Array(SEValue(rights)))
   }
 
+  final case class EnablePackages(
+      packages: List[ScriptLedgerClient.ReadablePackageId],
+      stackTrace: StackTrace,
+      continue: SValue,
+  ) extends Cmd {
+    override def description = "enablePackages"
+    override def execute(env: Env)(implicit
+        ec: ExecutionContext,
+        mat: Materializer,
+        esf: ExecutionSequencerFactory,
+    ): Future[SExpr] =
+      for {
+        client <- Converter.toFuture(env.clients.getParticipant(None))
+        _ <- client.enablePackages(packages)
+      } yield SEAppAtomic(SEValue(continue), Array(SEValue(SUnit)))
+  }
+
+  final case class DisablePackages(
+      packages: List[ScriptLedgerClient.ReadablePackageId],
+      stackTrace: StackTrace,
+      continue: SValue,
+  ) extends Cmd {
+    override def description = "disablePackages"
+    override def execute(env: Env)(implicit
+        ec: ExecutionContext,
+        mat: Materializer,
+        esf: ExecutionSequencerFactory,
+    ): Future[SExpr] =
+      for {
+        client <- Converter.toFuture(env.clients.getParticipant(None))
+        _ <- client.disablePackages(packages)
+      } yield SEAppAtomic(SEValue(continue), Array(SEValue(SUnit)))
+  }
+
+  final case class ListPackages(
+      stackTrace: StackTrace,
+      continue: SValue,
+  ) extends Cmd {
+    override def description = "listPackages"
+    override def execute(env: Env)(implicit
+        ec: ExecutionContext,
+        mat: Materializer,
+        esf: ExecutionSequencerFactory,
+    ): Future[SExpr] =
+      for {
+        client <- Converter.toFuture(env.clients.getParticipant(None))
```

Comment:
We only have 1 participant with IdeLedgerClient, but if we intend to use this outside of that (unsure), we might want to either specify the participant, or run on all participants